.bit/responses/1.4-Week 1 Step 4.md

```diff
@@ -11,17 +11,17 @@ Week 1 Step 4 ⬤⬤⬤⬤◯◯◯◯◯ | 🕐 Estimated completion: 35-45 min
 # Building our first function ⚡[HackerVoice]
 
 > Managing a server is pretty complicated. As a student, mastering serverless functions can help you to build projects that solve real-world problems by integrating APIs, constructing user interfaces, and analysing data without worrying about managing servers.
-> 
+>
 > [⚡️ 10 Ways to Use Serverless Functions](https://dev.to/aws/10-ways-to-use-serverless-functions-bme)
 
 ## ✅  Tasks:
-- [ ] Run `git pull` and create a new branch called `hackervoice` 
-- [ ] ***1:*** Create an Azure account 
-- [ ] ***2:*** Set up Azure locally on VS Code 
-- [ ] ***3:*** Create an HTTP trigger Azure function that gets content from a request parameter called `password` and returns it in the function's body 
-- [ ] ***4:*** Test your function locally with Postman and, when you confirm that it works, deploy your function 
-- [ ] 🚀 Place your function URL from the Azure portal in a [repository secret](https://docs.github.com/en/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository) called `HACKERVOICE_ENDPOINT` and commit the function's code in a file named `hackervoice/index.js` on the `hackervoice` branch 
-- [ ] 🚀 Create a pull request that merges `hackervoice` to `main`, but **do not merge it** 
+- [ ] Run `git pull` and create a new branch called `hackervoice`
+- [ ] ***1:*** Create an Azure account
+- [ ] ***2:*** Set up Azure locally on VS Code
+- [ ] ***3:*** Create an HTTP trigger Azure function that gets content from a request parameter called `password` and returns it in the function's body
+- [ ] ***4:*** Test your function locally with Postman and, when you confirm that it works, deploy your function
+- [ ] 🚀 Place your function URL from the Azure portal in a [repository secret](https://docs.github.com/en/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository) called `HACKERVOICE_ENDPOINT` and commit the function's code in a file named `hackervoice/index.js` on the `hackervoice` branch
+- [ ] 🚀 Create a pull request that merges `hackervoice` to `main`, but **do not merge it**
```

Comment:
I know this is same as before but I think it should say 'do not merge it yet' to show it will be merged at some point.

Reply:
No problem I'll update that. My question though is at what point do we no longer say do not merge? We as the user should not be merging until we see the gif TIME TO MERGE given by the bot.

Reply:
Where exactly do you want me to add that? & Should we use the same language everywhere?
W1S3 - does NOT say "do not merge"
W1S4 - already has the statement do not merge it
W1S5 - has the last line as ❗ Do not merge the branch!!

Reply:
@creaturenex Good point, we should be more consistent about that. Also, we are adding a feature to the bot that doesn't let them merge until the bot approves it. Here's what we can do: put the key words "do not merge this branch until the bot tells you to do so!" every time we ask students to make a pull request.

Reply:
Happy to approve once the minor changes are made

Reply:
I'ill get to this later today thank you for responding